src/main/java/de/epiceric/shopchest/utils/ShopUtils.java

```diff
@@ -18,6 +18,7 @@
 public class ShopUtils {
 
     private final HashMap<Location, Shop> shopLocation = new HashMap<>();
+    private final Collection<Shop> shopLocationValues = Collections.unmodifiableCollection(shopLocation.values());
```

Comment:
Does this automatically update with `shopLocation` when adding or removing a shop?